datadog-oci-orm/metrics-setup/schema.yaml

```diff
@@ -37,10 +42,16 @@ variables:
     default: datadog-metrics
   create_vcn:
     title: Create VCN
-    description: Optional variable to create virtual network for the setup. Otherwise, choose an existing subnet from VCN
+    description: Create virtual network for the function setup. Otherwise, choose an existing subnet from VCN.
     type: boolean
     default: true
 
+  compartment_ocid:
+    # prepopulates available values for compartment
+    required: true
+    type: oci:identity:compartment:id
+    description: The compartment OCID to where the terraform stack is created and the resources deployed
```

Comment:
description: The compartment OCID to where this terraform stack is created and the following networking resources, function, and connecter hub will be deployed.